deploy/manifests/sample-deployments/kong-enterprise-hybrid/kong-common.env

```diff
@@ -0,0 +1,18 @@
+KONG_PORTAL_GUI_HOST=http://192.168.1.100:30803
+KONG_PORTAL_GUI_LISTEN=0.0.0.0:8003, 0.0.0.0:8446 ssl
+KONG_PROXY_LISTEN=0.0.0.0:8000, 0.0.0.0:8443 ssl http2
+KONG_STATUS_LISTEN=0.0.0.0:8100
+KONG_ADMIN_GUI_AUTH=basic-auth
+KONG_ENFORCE_RBAC=on
+KONG_NGINX_WORKER_PROCESSES=1
+KONG_ADMIN_ACCESS_LOG=/dev/stdout
+KONG_ADMIN_ERROR_LOG=/dev/stderr
+KONG_PROXY_ERROR_LOG=/dev/stderr
+KONG_PROXY_ACCESS_LOG=/dev/stdout
+KONG_DATABASE=postgres
+KONG_PG_USER=kong
+KONG_PG_PASSWORD=kong
+KONG_PG_HOST=postgres
+KONG_PG_PORT=5432
+KONG_PASSWORD=123456
```

Comment:
Similar comment to what I put on the cert/key pair: this needs to be handled in documentation only, ideally with an obvious `KONG_PASSWORD=<YOU MUST CHANGE THIS VALUE>` call to action to try and avoid, as best possible, accidental creation of environments where the actual super-admin password is indeed `123456`.

Reply:
I changed all the exaxmples to `change-me`